### ISSUE TITLE
Support for CSS @apply Rule - Error: no writecb in Transform class

### ISSUE THREAD

**Issue by JosefJezek** [cssnext/cssnext#213]
How to resolve this error? We need for Polymer Themes.

- http://tabatkins.github.io/specs/css-apply-rule/
- https://www.polymer-project.org/1.0/docs/devguide/styling.html#custom-css-mixins
- https://github.com/StartPolymer/polymer-starter-kit-plus

I define mixin using `@apply` rule for `<paper-menu>` element.

```css
.navigation {
  --paper-menu-selected-item: {
    color: var(--default-primary-color);
  };
}
```
 -> `Error: no writecb in Transform class`

Thank you

**Comment by MoOx** [comment]
`@apply` is not yet supported yet in cssnext. We need to [upgrade to postcss v5](202) since https://github.com/pascalduez/postcss-apply need v5 parser.
Closing in favor of #203

**Comment by JosefJezek** [comment]
'@apply' rule is implemented in Polymer using JS. How to pass or ignore this error by parse css?

- https://www.polymer-project.org/1.0/docs/devguide/styling.html#custom-css-mixins

**Comment by MoOx** [comment]
You need to wait for #202 for now which is blocked by https://github.com/iamvdo/pleeease-filters/pull/12

**Comment by wincinderith** [comment]
I've still got the issue with the latest version.. Any updates on resolving this?
